### ISSUE TITLE
Skeletal animation problem

### ISSUE THREAD

**Issue by RisaStenr** [SaschaWillems/Vulkan#512]
I tried to run the skeletalanimation project, the floor loads fine but you do not see an animated character.  If you zoom in a bit you can see a bunch of random polygons on the screen that move but nothing that looks like a character.

**Comment by SaschaWillems** [comment]
Please add some details. E.g. GPU, driver, platform. And also check for validation layer messages.

**Comment by SaschaWillems** [comment]
Can you please check with a pre-built binary: https://vulkan.gpuinfo.org/downloads/skeletalanimation.7z

I can't reproduce this, the sample works fine for me on AMD, Intel, NVIDIA, ARM, Adreno, and windows, linux and android. So it may be a problem with your build environment.

**Comment by RisaStenr** [comment]
OK, i see the model and animation with the pre built.  So that means the problem is probably with visual studio, or what else could it be?  Any suggestions with how to figure this out is welcome, i'm pretty new to all this.

**Comment by RisaStenr** [comment]
So I have gotten the examples working in Eclipse on Ubuntu and I still don't see the character animation, at least on Nvidia.  I do see the ground animation though.  I will test in Eclipse on Ubuntu with Radeon and report back.

**Comment by RisaStenr** [comment]
Well, I tried this again on a fresh win 10 OS install on my AMD system and it worked!  I am not sure what the conflict was, as I had the bug in Ubuntu as well, at first I thought it had to be chrome as that was all I had in common between the two systems, but that wasn't it.  

Maybe the latest windows update fixed it?  I still had the bug on my Nvidia system yesterday, but I downloaded another windows update on my AMD i dont think I had on my nvidia...really not sure here.  Anyway, hooray! : )
